src/c_kzg_4844.h

```diff
@@ -93,10 +93,11 @@ typedef Bytes48 KZGProof;
  * The common return type for all routines in which something can go wrong.
  */
 typedef enum {
-    C_KZG_OK = 0,  /**< Success! */
-    C_KZG_BADARGS, /**< The supplied data is invalid in some way. */
-    C_KZG_ERROR,   /**< Internal error - this should never occur. */
-    C_KZG_MALLOC,  /**< Could not allocate memory. */
+    C_KZG_OK = 0,     /**< Success! */
+    C_KZG_BADARGS,    /**< The supplied data is invalid in some way. */
+    C_KZG_ERROR,      /**< Internal error - this should never occur. */
+    C_KZG_MALLOC,     /**< Could not allocate memory. */
+    C_KZG_BAD_VERIFY, /**< KZG proof verification failed. */
```

Comment:
I think `BAD_ARGS` is a good idea. However, please note that removing or reordering makes the underlying integers change which will probably make the new version of c-kzg-4844 not backward-compatible.

Reply:
I agree this change is a good idea; I tried to do it but it actually bloats the PR in a non-trivial way. I suggest we open another issue for this and do it at a later stage, so that it doesn't delay the review of this PR.

(currently working on the bindings)

Reply:
Agreed, that makes sense. Let's do this in another PR.